--- SConstruct.old	2012-11-07 23:05:01.525205324 +0000
+++ SConstruct	2012-11-07 23:37:13.123224865 +0000
@@ -164,7 +164,7 @@
 install_perms('$prefix/share/applications', 'desktop/mypaint.desktop')
 
 # location for achitecture-dependent modules
-install_perms('$prefix/lib/mypaint', module)
+install_perms('$prefix/lib64/mypaint', module)
 
 # Program and supporting UI XML
 install_perms('$prefix/bin', 'mypaint', perms=0755)
